package.json

```diff
@@ -97,6 +97,7 @@
     "chalk": "^5.3.0",
     "cheerio": "^1.0.0-rc.12",
     "copyfiles": "^2.4.1",
+    "cross-env": "^7.0.3",
```

Comment:
Any reason this dependency was added?

Reply:
Yeah it is used in our project to be able to for example run some tests (the Lit component tests in my case)

Reply:
I think it is only used in the `e2e` directory which is why we have it defined [here](https://github.com/webdriverio/webdriverio/blob/3670a36edb2f873d6797781ddc63fe4e5c5fca96/e2e/package.json#L48)